pkg/kinflate/util/manifestloader_test.go

```diff
@@ -62,21 +62,21 @@ func TestLoadNotExist(t *testing.T) {
 	fakeFS.Mkdir(".", 0644)
 	fakeFS.Create(badSuffix)
 	loader := kutil.ManifestLoader{FS: fakeFS}
-	_, err := loader.Read("Kube-manifest.yaml")
+	_, err := loader.MakeValidManifestPath("Kube-manifest.yaml")
 	if err == nil {
 		t.Fatalf("expect an error")
 	}
 	if !strings.Contains(err.Error(), "Run `kinflate init` first") {
```

Comment:
Not your fault, but relying on exact output string for errors makes is creating a maintenance burden.